object.feature

```diff
@@ -10,6 +10,10 @@ Feature: the object feature
     When copy object with key "<key>"
     Then copy object status code is 201
 
+    # Modify Object with metadata
```

Comment:
Let's add a new scenario for metadata test.

```
When create object ...
Then creat object ....
And get object metadata is ....
```

Reply:
OK